pyproject.toml

```diff
@@ -61,6 +61,13 @@ verify      = ["black --check .",
                "isort . --check-only",
                "ruff ."]
 
+[tool.hatch.envs.sqlglot]
```

Comment:
the environment has to be called something like `sqlglot-latest` to represent a difference from the direct dependency

Reply:
Done